src/java/org/apache/cassandra/db/marshal/UserType.java

```diff
@@ -215,7 +215,7 @@ public ByteBuffer serializeForNativeProtocol(Iterator<Cell<?>> cells)
         while (components.size() < size())
             components.add(null);
 
-        return pack(components, ByteBufferAccessor.instance);
+        return pack(components);
```

Comment:
when i did the last rebase this change got added else the history got too hard to work with; so rolling back